src/components/Person/Person.jsx

```diff
@@ -1 +1,14 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = person;
+  const partnerWith = sex === 'm' ? 'wife' : 'husband';
```

Comment:
Use more descriptive variable names to make the code clearer. 
For example, `partnerRole` can be used instead of `partnerWith`